test/BoardView.cpp

```diff
@@ -246,3 +248,80 @@ TEST_CASE("BoardView RGB444 cvt", "[BoardView]") {
 
     REQUIRE(br.stop());
 }
+
+/*
+ * rgb888ToRgb565
+ * 76543210 | 76543210
+ * GGGBBBBB   RRRRRGGG
+ */
+TEST_CASE("BoardView RGB565 Read Test", "[BoardView]") {
+    // convert three bytes(24bits) of rgb888 to two bytes(18bits) of rgb565
+    std::byte res[2];
+    std::byte red_bits[] = {0xFF_b, 0_b, 0_b};
+    std::byte green_bits[] = {0_b, 0xFF_b, 0_b};
+    std::byte blue_bits[] = {0x0_b, 0_b, 0xFF_b};
+    std::byte black_bits[] = {0_b, 0_b, 0_b};
+    std::byte white_bits[] = {0xFF_b, 0xFF_b, 0xFF_b};
```

Comment:
Use more varied values, else it we might as well only check that plain colors work

Reply:
I am gonna add an elegant purple or the shining green!